### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by rarkins** [deanmalmgren/flo#71]
@deanmalmgren I like the goals and approach of the project, but see it hasn't been updated for over a year, so I hesitate to use it. How would you describe the state of the project and its continuity?

**Comment by deanmalmgren** [comment]
Hey @rarkins, Thanks for the note and your interest in using `flo`. I paused active development on this project for two reasons.

For one thing, I wanted to try using it a few times before I over-developed something (I had a lot of ideas for cool features to incorporate). I haven't publicized this project at all, so it isn't robustly tested in the real world as far as I know. Having said that, I've used flo on a handful of projects (as has @gabegaster) and found it quite useful and intuitive, but perhaps I'm biased? I'd greatly appreciate any feedback you have about it and, as much as I can, I'd try and improve it based on your feedback and suggestions.

The second reason I paused development is that I became aware of [luigi](https://github.com/spotify/luigi) which is very actively maintained by the Spotify team and beyond and it appears to have very similar aims. I haven't used luigi at all so I can't speak to whether its a better choice and I should just deprecate flo, but I'd of course welcome your perspective.

Does that answer your question?
